ee/clickhouse/views/test/test_clickhouse_event.py

```diff
@@ -44,3 +44,9 @@ def test_optimize_query(self, patch_sync_execute):
         patch_sync_execute.return_value = [("event", "d", "{}", timezone.now(), "d", "d", "d") for _ in range(0, 100)]
         response = self.client.get("/api/event/").json()
         self.assertEqual(patch_sync_execute.call_count, 3)
+
+    @patch("ee.clickhouse.views.events.sync_execute")
```

Comment:
This test belongs under postgres event view tests - same behavior should occur for both rather than the endpoints diverging in how they handle things.